packages/css/input.css

```diff
@@ -180,7 +185,6 @@
   --dsc-input-affix-border: 1px solid var(--ds-color-neutral-border-default);
   --dsc-input-affix-padding-inline: var(--ds-spacing-4);
 
-  align-items: center;
```

Comment:
Why is this removed? It has cause the affixes to not be aligned anymore
![image](https://github.com/user-attachments/assets/d55bd1a6-8f1f-4d43-afae-9376f860069e)

Reply:
Yupp, saw it in chromatic now 🙈 removed wrong one :D re-adding